lms/djangoapps/shoppingcart/tests/test_models.py

```diff
@@ -265,7 +265,7 @@ def test_purchase(self):
                     }
                 ]
             },
-            context={'Google Analytics': {'clientId': None}}
+            context={'ip': None, 'Google Analytics': {'clientId': None}}
```

Comment:
@peter-fogg In the above cases we are actually making a client request so the local IP is used.  In this particular test case we are not making any request, we are just making calls on the model directly.